<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>756</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文琥珀</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:#DCDCDC;
border-right:1px solid black;
border-left:1px solid black;
border-top:1px solid black;
border-bottom:1px solid black;
border-top-left-radius:10px;
border-bottom-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:10px;
}
QPushButton{
    border:none;
    border-bottom:1px solid black;
    font-size:18px;
    font-weight:700;
 }
QPushButton:hover{border-left:4px solid grey;font-weight:700;}
QCheckBox{
border:none}
</string>
  </property>
  <widget class="QGroupBox" name="DbGRoup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>数据管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="DbStateGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>数据库状态</string>
    </property>
    <widget class="QPushButton" name="initdb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
       <pointsize>-1</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>数据库初始化</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>171</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="UserNumberverticalLayout">
      <item>
       <widget class="QLabel" name="UserNumberLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
         </font>
        </property>
        <property name="text">
         <string>已存人脸样本数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="UserlcdNumber"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="trainGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>训练数据</string>
    </property>
    <widget class="QPushButton" name="trainButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
       <pointsize>-1</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>开始训练</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="equalizeHistCheckBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="text">
      <string>直方图均衡</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="logGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>系统日志</string>
    </property>
    <widget class="QTextEdit" name="logtextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="facegroupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>581</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{border:none}</string>
   </property>
   <property name="title">
    <string>人脸库</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabelWiget{background:#DCDCDC}</string>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column>
     <property name="text">
      <string>学号</string>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="background">
      <color>
       <red>220</red>
       <green>220</green>
       <blue>220</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Face ID</string>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="background">
      <color>
       <red>220</red>
       <green>220</green>
       <blue>220</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="background">
      <color>
       <red>220</red>
       <green>220</green>
       <blue>220</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>英文名</string>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="background">
      <color>
       <red>220</red>
       <green>220</green>
       <blue>220</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>注册时间</string>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
      </font>
     </property>
     <property name="background">
      <color>
       <red>220</red>
       <green>220</green>
       <blue>220</blue>
      </color>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border:none}</string>
    </property>
    <property name="text">
     <string>注：Face ID为 -1 说明该用户的人脸数据未被训练</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="deleteUserGroupBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>70</y>
     <width>251</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>删除用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="queryUserGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>学号查询</string>
    </property>
    <widget class="QLineEdit" name="QuerylineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="QueryButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文中宋</family>
       <pointsize>-1</pointsize>
       <weight>87</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>开始查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>221</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border:none}
QGroupBox{border:none}</string>
    </property>
    <property name="title">
     <string>查询结果</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="usernumberLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>学号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UsernumlineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="usernameLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>姓名：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UsernamelineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="FaceIDLabel">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Face_ID：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="FaceIDLineedit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="deleteUserButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>-1</pointsize>
      <weight>87</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>删除用户</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="queryResultLabel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>700</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_close">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_visit">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="left_mini">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
